src/main/java/cms/gov/madie/measure/services/MeasureService.java

```diff
@@ -862,6 +862,18 @@ public List<Measure> getQiCoreMeasuresByCmsId(Integer qdmCmsId) {
   }
 
   public void validateCmsIdAssociation(String username, Measure qiCoreMeasure, Measure qdmMeasure) {
+    if (!qiCoreMeasure.getModel().equals(ModelType.QI_CORE.getValue())
```

Comment:
Could you add some null checks on the measure objects please? I suspect its already done at the controller level, but this is a public method so other classes might call it and we should verify assumptions about its inputs.